keyboards/pitta/sq80/sq80.c

```diff
@@ -0,0 +1,47 @@
+#include "sq80.h"
+#include "led.h"
+
+void matrix_init_kb(void) {
+  // Keyboard start-up code goes here
+  // Runs once when the firmware starts up
+  matrix_init_user();
+  led_init_ports();
+};
+
+void matrix_scan_kb(void) {
+  // Looping keyboard code goes here
+  // This runs every cycle (a lot)
+  matrix_scan_user();
+};
+
+
+void led_init_ports(void) {
+  // Set LED pin as output
+  DDRD |= (1 << 0);
+  DDRD |= (1 << 1);
+  DDRD |= (1 << 2);
+  // Default to off
+  PORTD |= (1 << 0);
+  PORTD |= (1 << 1);
+  PORTD |= (1 << 2);
```

Comment:
This should be:
```c
setPinOutput(D0);
setPinOutput(D1);
setPinOutput(D2);
writePinHigh(D0);
writePinHigh(D1);
writePinHigh(D2);
```

Reply:
Don't need the writePin here, either.   The led_update_kb function should be called soon after this, and set th pins